### PR TITLE
[JENKINS-52242] Create changelog for remoting-kafka-plugin

### PR DESCRIPTION
**Pull request by pvtuan10**
JIRA task: [JENKINS-52242](https://issues.jenkins-ci.org/browse/JENKINS-52242)

@oleg-nenashev @Supun94

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Changelog
+===
+
+# 1.0.0-alpha-1-SNAPSHOT
```
Comment (oleg-nenashev) [inline CHANGELOG.md]: Remove "-SNAPSHOT"?

**Comment by pvtuan10** [comment]
@oleg-nenashev I changed to "Initial release".

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Features:
+
+* [JENKINS-51414](https://issues.jenkins-ci.org/browse/JENKINS-51414) - Implementation of command transport for command invocation (for master).
+* [JENKINS-51708](https://issues.jenkins-ci.org/browse/JENKINS-51708) - Implementation of command transport for command invocation (for agents).
+* [JENKINS-51470](https://issues.jenkins-ci.org/browse/JENKINS-51470) - Remoting Kafka agents should provide connection security.
+* [JENKINS-51857](https://issues.jenkins-ci.org/browse/JENKINS-51857) - Stop storing Kafka Properties on disk.
+* [JENKINS-52190](https://issues.jenkins-ci.org/browse/JENKINS-52190) - Update Remoting Kafka to the released versions of Remoting/Core with extensibility API.
+* [JENKINS-51942](https://issues.jenkins-ci.org/browse/JENKINS-51942) - Improve Kafka producer-consumer topic model.
+
+Bugfixes:
```
Comment (oleg-nenashev) [inline CHANGELOG.md]: Remove bugfixes? It's an initial release, so no known bugs

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+Changelog
+===
+
+# Initial release
```
Comment (oleg-nenashev) [inline CHANGELOG.md]: 1.0.0-alpha-1. "Initial release" was supposed to be in the text, but features are fine

**Pull request merged**